README.rst

```diff
@@ -1,19 +1,72 @@
-|Build Status| |Codecov| |license|
-
-#############################
 frontend-app-course-authoring
 #############################
 
-Please tag `@edx/teaching-and-learning <https://github.com/orgs/edx/teams/teaching-and-learning>`_ on any PRs or issues.  Thanks.
+|license-badge| |status-badge| |codecov-badge|
 
-************
-Introduction
-************
+
+Purpose
+*******
 
 This is the Course Authoring micro-frontend, currently under development by `2U <https://2u.com>`_.
 
 Its purpose is to provide both a framework and UI for new or replacement React-based authoring features outside ``edx-platform``.  You can find the current set described below.
 
+
+************
+Getting Started
+************
+
+Prerequisites
+=============
+
+The `devstack`_ is currently recommended as a development environment for your
+new MFE.  If you start it with ``make dev.up.lms`` that should give you
+everything you need as a companion to this frontend.
+
+Note that it is also possible to use `Tutor`_ to develop an MFE.  You can refer
+to the `relevant tutor-mfe documentation`_ to get started using it.
+
+.. _Devstack: https://github.com/openedx/devstack
+
+.. _Tutor: https://github.com/overhangio/tutor
+
+.. _relevant tutor-mfe documentation: https://github.com/overhangio/tutor-mfe#mfe-development
+
+Cloning and Startup
+===================
+
+In the following steps, replace "[PLACEHOLDER]" with the name of the repo you
+created when copying this template above.
+
+
+1. Clone your new repo:
+
+  ``git clone https://github.com/openedx/frontend-app-course-authoring.git``
+
+2. Use node v18.x.
+
+   The current version of the micro-frontend build scripts support node 18.
+   Using other major versions of node *may* work, but this is unsupported.  For
+   convenience, this repository includes an .nvmrc file to help in setting the
+   correct node version via `nvm <https://github.com/nvm-sh/nvm>`_.
```

Comment:
I think replacing this with instructions to run `nvm use` would be enough.  (Provided this repo has an `.nvmrc`, which seems to be the case.)